### PR TITLE
Create header and headline

### PR DESCRIPTION
**Pull request by Lornakaboro**
This is a portfolio that is aimed at showcasing my accomplishments  ,skills , educational and work backgrounds.
I created  index.html, style.css and READ.me files.
I also added a background and social media images from Figma.

### Getting Started

To get a local copy up and running follow these simple example steps.

### Prerequisites

- Git installed in desktop
- Code editor of your choice i.e; Visual Studio Code
- Browser of your choice i.e; Mozilla Firefox
- Terminal of your choice i.e; Git Bash

### Setup

To get a local copy up and running follow these simple example steps.
- use the git clone command with this [link](git@github.com:Lornakaboro/my-portfolio.git)
- cd into my-portfolio
- Switch branch using this command `git checkout create-header-and-headline`
- Open index.html in your browser
- You will be able to see a template with my name, a brief description and social media handles

Screenshot 2022-08-19 at 17 56 12

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <link rel="stylesheet" href="style.css">
+  <title>My Portfolio</title>
+  <script src="https://kit.fontawesome.com/c6cb015206.js" crossorigin="anonymous"></script>
+<body>
+
+  <div class="intro-section">
+    <div class="header">
+      <h3 class="logo">My Logo</h3>
+      <i class="fa fa-bars" aria-hidden="true"></i>
+    </div>
```
Comment (mahbubul14) [inline index.html:16]: - [ ] Instead of a `div` tag it should be wrapped in a `header` tag. Kindly work on it. :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>My Portfolio</title>
+  <script src="https://kit.fontawesome.com/c6cb015206.js" crossorigin="anonymous"></script>
+<body>
+
+  <div class="intro-section">
+    <div class="header">
+      <h3 class="logo">My Logo</h3>
```
Comment (mahbubul14) [inline index.html:14]: - [ ] The logo ("My logo") on the left should be in `a` tag. :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        a project you need coded, don&#39;t hestiate to contact me.</p>
+      <h4 class="connect">LET&#39;S CONNECT</h4>
+      <div>
+        <img class="icon" src="./twitter.png" alt="twitter-icon">
+        <img class="icon" src="./linkedin.png" alt="linkedin-icon">
+        <img class="icon" src="./medium.png" alt="medium-icon">
+        <img class="icon" src="./github.png" alt="github-icon">
+        <img class="icon" src="./hand.png" alt="hand-icon">
+      </div>
+    </div>
```
Comment (mahbubul14) [inline index.html:31]: - [ ] This section should be wrapped in a `section` tag. :+1:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        Look through some of my work and experience! If you like what you see and have 
+        a project you need coded, don&#39;t hestiate to contact me.</p>
+      <h4 class="connect">LET&#39;S CONNECT</h4>
+      <div>
+        <img class="icon" src="./twitter.png" alt="twitter-icon">
+        <img class="icon" src="./linkedin.png" alt="linkedin-icon">
+        <img class="icon" src="./medium.png" alt="medium-icon">
+        <img class="icon" src="./github.png" alt="github-icon">
+        <img class="icon" src="./hand.png" alt="hand-icon">
+      </div>
```
Comment (mahbubul14) [inline index.html:30]: - [ ] The list of icons should be in a ul tag to group them, and li per icon; each icon has a link (a tag). :+1:

File: style.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  margin: 0;
+  padding: 20px;
+  font-weight: 700;
+  font-style: bold;
+}
+
+.info-box {
+  padding: 24px;
+  margin-top: 90px;
+}
```
Comment (mahbubul14) [inline style.css:43]: - [ ] The content should be aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. Kindly work on it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Lornakaboro ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] Please use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version. It is not currently optimized for the mobile version.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 6]
```diff
@@ -1 +1,55 @@
...
+- use the git clone command with this [link](git@github.com:Lornakaboro/my-portfolio.git)
+- cd into my-portfolio
+- Switch branch using this command `git checkout create-header-and-headline`
+- Open index.html in your browser
+- You will be able to see a template with my name, a brief description and social media handles.
+
+## Authors
+
+👤 **Author1**
+
```
Comment (emmiiorji) [inline README.md:35]: _[Optional]_ I recommend that you replace "Author1" with your name because you'll agree that it suggests so.

**Review by emmiiorji — APPROVED** [review]
Hi @Lornakaboro, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

goodjob-slowclap


## Good Highlights ✅

- Requested changes have been implemented
- Good commit messages
- Linter checks are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:

- organizing your images into a folder so as to cluttering the root folder.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**